hieradata/class/integration/jenkins.yaml

```diff
@@ -2,6 +2,7 @@
 govuk_jenkins::config::executors: '4'
 
 govuk_jenkins::job_builder::jobs:
+  - govuk_jenkins::jobs::copy_data_from_integration_to_aws
```

Comment:
I don't think we need this, because Integration is already in AWS, so it would be copying to itself.

Reply:
Correct, done.